lib/suse/connect/hwinfo/arm64.rb

```diff
@@ -0,0 +1,62 @@
+# Collect hardware information for aarch64 systems
+class SUSE::Connect::HwInfo::ARM64 < SUSE::Connect::HwInfo::Base
+  class << self
+    def hwinfo
+      {
+        hostname: hostname,
+        cpus: cpus,
+        sockets: sockets,
+        hypervisor: hypervisor,
+        arch: arch,
+        uuid: uuid
+      }
+    end
+
+    def cpus
+      output['CPU(s)'].to_i
+    end
+
+    def sockets
+      output['Socket(s)'].to_i
+    end
+
+    def hypervisor
+      vendor = execute('systemd-detect-virt -v', false, [0, 1])
```

Comment:
> Did you try this out? In which case it returns with exit code 1? 

From its manpage:
```
EXIT STATUS
       If a virtualization technology is detected, 0 is returned, a non-zero code otherwise.
```
According to Dirk and my machine, it always exits with `1` when it returns `none`.

> Can I login to that instance to see?

Sure! I asked Dirk to keep the setup running until our Review Meeting at least.